pkg/tsdb/cloudmonitoring/cloudmonitoring.go

```diff
@@ -206,6 +206,22 @@ func newInstanceSettings(httpClientProvider httpclient.Provider) datasource.Inst
 	}
 }
 
+func migrateMetricTypeFilter(metricTypeFilter string, prevFilters interface{}) []string {
+	metricTypeFilterArray := []string{"metric.type", "=", metricTypeFilter}
+	filters := []string{}
```

Comment:
Nit: can we name the return variable `filters` , e.g. `(filters []string)` and then we don't need this here?

Reply:
generally I don't like named return variables very much. I found them more difficult to read in bigger functions and prone to bugs (e.g. returning partial results with errors) :thinking:

Reply:
Works for me :smile: I tend to use them in cases like this where I'm creating a variable to return it later. Like I said though, it's a nit.